xbmc/music/windows/GUIWindowMusicNav.cpp

```diff
@@ -253,6 +254,138 @@ std::string CGUIWindowMusicNav::GetQuickpathName(const std::string& strPath) con
   }
 }
 
+bool CGUIWindowMusicNav::ManageInfoProvider(const CFileItemPtr &item)
```

Comment:
parameter should be either `const CFileItem& item` or `CFileItemPtr item` taking a const ref to std::shared_ptr should be avoided as it's easy to misunderstand the different reference semantics.